src/header-navigation/README.md

```diff
@@ -61,22 +61,28 @@ export default (props) => <HeaderNavigation
 
 ## `HeaderNavigation` API
 
-* `children: React$Node` - Required.
-  All the children of header navigation. It accepts any components, but it is useful to use `NavigationList` and `NavigationItem` to align and follow style guides.
-  
+* `children: React.Node`
+  * All the children of header navigation. It accepts any components, but it is useful to use `NavigationList` and `NavigationItem` to align and follow style guides.
+
 ## `StyledNavigationList` API
 
-* `align: ALIGN.right | ALIGN.left | ALIGN.center` - Optional.
-  Alignment of elements inside of navigation list
-* `children: React$Node` - Required.
-  All the children of NavigationList.  
-  
+* `align: $Values<ALIGN> = undefined`
```

Comment:
Should it be nullable type and the default set to `null`?

Reply:
Feels like the ` ALIGN.right | ALIGN.left | ALIGN.center` desription is more user-friendly, especially when someone looking at the docs doesn't use/not familiar with flow types.

Reply:
yeah i agree it get's rid of some readability. we were not very consistent across rfcs in the pattern, so went with flow conventions. Although, i want some syntax to describe a set of constants (a pattern that comes up often). This would make maintaining rfcs in the future

Reply:
do you have a suggestion for an alternative syntax?

Reply:
I see, agree about maintainability and a single pattern for constants. Maybe something like `ALIGN[key]` or `ALIGN[string]`?

Reply:
or `ALIGN[Object.keys(ALIGN)]`